msal/src/test/java/com/microsoft/identity/nativeauth/NativeAuthPublicClientApplicationKotlinTest.kt

```diff
@@ -101,10 +112,31 @@ class NativeAuthPublicClientApplicationKotlinTest : PublicClientApplicationAbstr
     private val password = "verySafePassword".toCharArray()
     private val code = "1234"
     private val emptyString = ""
+    private val infoPIIToCheck = listOf(
```

Comment:
nit: why is this variable called `info`?

Reply:
Short for information. After finding out that PII = Personally identifiable information, I renamed the variable as PIIToCheck.